aten/src/ATen/native/native_functions.yaml

```diff
@@ -3475,28 +3475,35 @@
 
 - func: std(Tensor self, bool unbiased=True) -> Tensor
   variants: function, method
-  dispatch:
-    CPU, CUDA: std
 
 - func: std.dim(Tensor self, int[1] dim, bool unbiased=True, bool keepdim=False) -> Tensor
   variants: function, method
+
+- func: std.correction(Tensor self, int[1]? dim, *, int? correction, bool keepdim=False) -> Tensor
+  variants: function, method
   dispatch:
     CPU, CUDA: std
 
 - func: std_mean(Tensor self, bool unbiased=True) -> (Tensor, Tensor)
   variants: function
-  dispatch:
-    CPU, CUDA: std_mean
 
 - func: std_mean.dim(Tensor self, int[1] dim, bool unbiased=True, bool keepdim=False) -> (Tensor, Tensor)
   variants: function
+
+- func: std_mean.correction(Tensor self, int[1]? dim, *, int? correction, bool keepdim=False) -> (Tensor, Tensor)
```

Comment:
These signatures look good, but I think they need to require correction `int correction` so they're only matched when `correction` is actually specified by the user

Reply:
`int?` without a default means the user needs to explicitly pass `int` or `None`. Either way, the user needs to specify it.

Reply:
Also, if we change to optional later when the default gets added then that requires a breaking change to the JIT schema so the "forward compatibility" issues @heitorschueroff mentions wouldn't be resolved.